imports/methods/announcements/post.ts

```diff
@@ -0,0 +1,38 @@
+import { check } from 'meteor/check';
+import { ValidatedMethod } from 'meteor/mdg:validated-method';
+import { Meteor } from 'meteor/meteor';
+import Ansible from '../../Ansible';
+import Announcements from '../../lib/models/Announcements';
+import MeteorUsers from '../../lib/models/MeteorUsers';
+import PendingAnnouncements from '../../lib/models/PendingAnnouncements';
+import { userMayAddAnnouncementToHunt } from '../../lib/permission_stubs';
+
+export default new ValidatedMethod({
+  name: 'Announcements.methods.post',
```

Comment:
Note that I've changed the name of the method here. This format matches the style of the [module documentation](https://github.com/meteor/validated-method#using-simpleschema) and feels a little cleaner to me. Their example also names the function (or in my case, module) after the last component of the string name.